index.html

```diff
@@ -12,28 +12,191 @@
 </head>
 
 <body>
-  <header>
-    <nav id="menu-top">
-      <ul class="topnav">
-        <li class="topnav-logo"><a href="#">My Logo</a></li>
-        <li class="topnav-right"><a href="#">Home</a></li>
-        <li class="topnav-right"><a href="#">About</a></li>
-        <li class="topnav-right"><a href="#">Portfolio</a></li>
-        <li class="dropdownIcon"><a href="#">&#9776;</a></li>
-      </ul>
-    </nav>
-  </header>
+  <section class="headersec">
+    <header>
+      <nav id="menu-top">
+        <ul class="topnav">
+          <li class="topnav-logo"><a href="#">My Logo</a></li>
+          <li class="topnav-right"><a href="#">Home</a></li>
+          <li class="topnav-right"><a href="#">About</a></li>
+          <li class="topnav-right"><a href="#">Portfolio</a></li>
+          <li class="dropdownIcon"><a href="#">&#9776;</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="about-flex-container">
+      <div class="name">
+        <h1>I’m Lilian
+          Glad to see you!</h1>
+      </div>
+      <div class="description">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+          and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p>
+      </div>
+      <div class="connect">
+        <a href="#" class="connect">LET’S CONNECT</a>
+      </div>
```

Comment:
Kindly fix the styling of the part to be similar to the original design shown below
#### Original design
![right1](https://user-images.githubusercontent.com/58376020/125164192-50616600-e191-11eb-860f-02375c684218.png)

#### Your Project
![wrong1](https://user-images.githubusercontent.com/58376020/125164202-59eace00-e191-11eb-80ce-277d646a4e9f.png)

Reply:
Resolved